### DISCUSSION TITLE
Example app based on Expo?

### DISCUSSION THREAD

**Discussion by RobJacobson** [rnmapbox/maps#3677] [Ideas]
Thanks for maintaining this package! I'm using rnmapbox with Expo. I found it difficult to get started, since the example app isn't based around the Expo workflow, but I'm figuring it out piece-by-piece.

Would there be any interest in an example app that's built with Expo in mind, including instructions for EAS Build? If so, I would be happy to take a stab at one. It would help me fill in the gaps in my knowledge, and it might be a good jumping-off point for others who are just getting started.

Also, I found it tricky to use the existing example app given its status as a monorepo with the parent code. That made it more difficult to get my head around the example code, separate from the main library code. Would there be an interest in refactoring that into a traditional project that links to the npm package? That would be cleaner and easier as a teaching guide, IMHO.

I'd appreciate any thoughts. Thanks in advance.

**Comment by mfazekas** [comment]
@RobJacobson hi, so example app is really a test app for development of rnmapbox maps library, not meant to be used as starting point for building an app with rnmapbox/maps. We should improve docs if it's not clear somewhere.

Yes we could make a minimal example app in another repo.

**Comment by benpretorius** [comment]
Hi there @RobJacobson 

Would be fantastic to hear if you’ve taken a stab at this! I’m currently also struggling a lot to find ways to implement this module specifically with the Expo workflow and EAS builds.
